src/util/debug.h

```diff
@@ -5,6 +5,16 @@
 
 #include "errordialoghandler.h"
 
+#ifdef MIXXX_DEBUG_ASSERTIONS_ENABLED
+#define DBG(value) \
+    (qDebug().nospace() << #value << " = " << (value) << " [" << __FILE__ << ":" << __LINE__ << "]")
```

Comment:
I think the real utility of rusts debug macro comes from the fact that it returns the value that has been passed in. That enables it to be inserted into any expression at will. The merged solution doesn't do that unfortunately, but it is achievable with a an IILE:
```suggestion
    ([](auto&& val) {
        qDebug().nospace() << #value << " = " << val << " [" << __FILE__ << ":" << __LINE__; << "]"; 
        return val;
    })(value)
```
Not sure how the types for this work out though.